packages/paste-core/components/topbar/stories/topbar.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import {Stack} from '@twilio-paste/stack';
+import type {StoryFn} from '@storybook/react';
+import {ProductFlexIcon} from '@twilio-paste/icons/esm/ProductFlexIcon';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {
+  Sidebar,
+  SidebarPushContentWrapper,
+  SidebarOverlayContentWrapper,
+  SidebarHeader,
+  SidebarCollapseButton,
+  SidebarCollapseButtonWrapper,
+} from '@twilio-paste/sidebar';
+
+import {Topbar} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Topbar',
+};
+
+const TonsOfContent: React.FC = () => {
+  const content = [];
+  for (let i = 0; i < 10; i++) {
+    content.push(<Paragraph key={i}>Dummy content</Paragraph>);
+  }
+  return <>{content}</>;
+};
+
+export const PushDefaultTopbar: StoryFn = () => {
```

Comment:
suggestion: can you add a "open sidebar" button to this story like you have in the sidebar stories? once it's collapsed there's no way to re-open it